src/miniflask/event.py

```diff
@@ -1,20 +1,24 @@
 import inspect
+from typing import Callable, List
+from dataclasses import dataclass
+from enum import Enum
 
 
-class outervar():
-    pass
+class outervar(Enum):
```

Comment:
I thought this through once more.
TBH, I do not see any advantage over the previous approach.

Maybe it makes more sense to define `outervar` like this:
```
class Outervar(Enum):
   outervar = "outervar"
outervar = Outervar.outervar
```

But actually I don't see any advantages over just

```
class outervar:
    pass
```
as this class is only intended to be a way of specifying a certain type of argument for an event.

The advantage for the latter one is that in the future this allows more special `outervar` implementations to appear in the code and it is also cleaner in my opinion.

Reply:
I agree, let's use a plain class and disable the error.